src/cluster/cluster.go

```diff
@@ -1,51 +1,62 @@
 package cluster
 
 import (
-	"errors"
-	"firecontroller/component"
 	"firecontroller/microcontroller"
 	mc "firecontroller/microcontroller"
 	"firecontroller/utilities"
 	"log"
-	"strconv"
 
 	"github.com/spf13/viper"
 )
 
 //Cluster - This object defines an array of microcontrollers
 type Cluster struct {
-	Name                  string
-	SlaveMicrocontrollers []mc.Microcontroller
-	Master                *mc.Microcontroller
-	Me                    *mc.Microcontroller
+	Name             string
+	Microcontrollers []mc.Microcontroller
```

Comment:
We discussed this offline not too long ago. A flag has been added to the micros to indicate if it is master. A func is added to replace the pointer to Master. Since the master and slaves all reside in 1 list, it can just be called Microcontrollers